src/components/MovieCover/MovieCover.css

```diff
@@ -1,5 +1,28 @@
 .movie-cover {
```

Comment:
Not critical for now so I'm going to approve this so that we can move forward, but removing max-height took care of the different widths but now instead of the wider covers going to the next line we have the heights not lining up 
<img width="891" alt="image" src="https://github.com/lauraguerra1/rancid-tomatillos/assets/123283073/7a3f0ad1-dff5-48c6-b6ad-838be7763f64">

Reply:
Oh! I didn't notice this! We can fix it with some media queries though